modules/gltf/src/lib/extensions/deprecated/EXT_feature_metadata.ts

```diff
@@ -110,6 +135,186 @@ function getPropertyDataFromBinarySource(
   return data;
 }
 
+/**
+ * Get properties from texture associated with all mesh primitives.
+ * @param scenegraph
+ * @param featureTextureProperty
+ * @param attributeName
+ * @returns Feature texture data
+ */
+function getPropertyDataFromTexture(
+  scenegraph: GLTFScenegraph,
+  featureTextureProperty: FeatureTextureProperty,
+  attributeName: string
+): number[] {
+  const json = scenegraph.gltf.json;
+  if (!json.meshes) {
+    return [];
+  }
+  const featureTextureTable: number[] = [];
+  for (const mesh of json.meshes) {
+    for (const primitive of mesh.primitives) {
+      processPrimitiveTextures(
+        scenegraph,
+        attributeName,
+        featureTextureProperty,
+        featureTextureTable,
+        primitive
+      );
+    }
+  }
+  return featureTextureTable;
+}
+
+// eslint-disable-next-line max-statements
+function processPrimitiveTextures(
+  scenegraph: GLTFScenegraph,
+  attributeName: string,
+  featureTextureProperty: FeatureTextureProperty,
+  featureTextureTable: number[],
+  primitive: GLTFMeshPrimitive
+): void {
+  /*
+    texture.index is an index for the "textures" array.
+    The texture object referenced by this index looks like this:
+    {
+    "sampler": 0,
+    "source": 0
+    }
+    "sampler" is an index for the "samplers" array
+    "source" is an index for the "images" array that contains data. These data are stored in rgba channels of the image.
+
+    texture.texCoord is a number-suffix (like 1) for an attribute like "TEXCOORD_1" in meshes.primitives
+    The value of "TEXCOORD_1" is an accessor that is used to get coordinates. These coordinates ared used to get data from the image.
+  */
+  const json = scenegraph.gltf.json;
+  const textureData: number[] = [];
+  const texCoordAccessorKey = `TEXCOORD_${featureTextureProperty.texture.texCoord}`;
+  const texCoordAccessorIndex = primitive.attributes[texCoordAccessorKey];
+  const texCoordBufferView = scenegraph.getBufferView(texCoordAccessorIndex);
+  const texCoordArray: Uint8Array = scenegraph.getTypedArrayForBufferView(texCoordBufferView);
+
+  const textureCoordinates: Float32Array = new Float32Array(
```

Comment:
We can get array type from `accessor`.`componentType` 
* https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#accessors
* https://github.com/visgl/loaders.gl/blob/fe610fc6ff83dac13787df35ff07b8aead24f1ed/modules/gltf/src/lib/gltf-utils/gltf-utils.ts#L24